ticket_facts.view.lkml

```diff
@@ -0,0 +1,243 @@
+view: ticket_facts {
+  derived_table: {
+    sql: SELECT
+          audits.ticket_id,
+          tickets.status,
+          tickets.via__channel,
+          tickets.via__source__rel,
+          tickets.via__source__to__name,
+          groups.name,
+          metrics.group_stations AS number_of_groups_involved,
+          COUNT(DISTINCT audits.id) FILTER(WHERE audits.via__channel = 'voice' AND audits.via__source__from__name = 'Getaround'
+                                  AND audits.via__source__rel = 'outbound')  AS number_outbound_calls,
+          COUNT(DISTINCT audits.id) FILTER(WHERE (audits.via__channel = 'voice' AND audits.via__source__to__name = 'Getaround'
+                                  AND audits.via__source__rel = 'inbound')
+                                  OR (audits.via__channel = 'voice' AND audits.via__source__rel = 'voicemail'))  AS number_inbound_calls,
+          COUNT(DISTINCT audits.id) FILTER(WHERE (events.type = 'Comment' AND events.public = true AND audits.via__source__to__name = 'Getaround')
+                                  OR (events.type = 'Comment' AND events.public = true AND audits.via__channel = 'mobile_sdk')
+                                  OR (events.type = 'Comment' AND events.public = true AND audits.via__source__rel = 'follow_up'))  AS number_inbound_emails,
```

Comment:
Currently, all audits w/ via__source__rel = 'follow_up' are from 'web' which is why I didn't bother to include it, but good catch, as I should include it in case that changes to make sure everything is consistent.  Updated.